docs/remote/devcontainer-cli.md

```diff
@@ -0,0 +1,54 @@
+---
+Order: 12
+Area: remote
+TOCTitle: devcontainer CLI
+PageTitle: Installing and working with the devcontainer CLI
+ContentId: 8946213d-716e-41ca-955f-944a41c70353
+MetaDescription: Documentation on using the devcontainer CLI with the Visual Studio Code Remote - Containers extension
+---
+# `devcontainer` Command Line Interface
+
+Remote-Containers has a `devcontainer` Command Line Interface (CLI) which allows you to interact with a dev container from your terminal.
+
+## Installing the devcontainer CLI
```

Comment:
We've gotten official guidance to use two words when referring to dev containers, so "dev container" instead of "devcontainer." I recognize for the CLI itself the command is one word (`devcontainer`), but wondering if in any other uses, we should stick with two words to follow the appropriate practice.

Reply:
So we should probably either refer to it as:
- `devcontainer` CLI
- dev container CLI

Reply:
Yeah, that was what I'd been trying to do - slipped up in a few places, but hopefully now all consistent 😄